src/improvements/tasks/sep/017-U16-opcm-upgrade-v400-base/config.toml

```diff
@@ -22,11 +22,11 @@ StandardValidatorV400 = "0xA8a1529547306FEC7A32a001705160f2110451aE" # Freshly d
 
 [stateOverrides]
 0x0fe884546476dDd290eC46318785046ef68a0BA9 = [
-     {key = "0x0000000000000000000000000000000000000000000000000000000000000005", value = 23}
+     {key = "0x0000000000000000000000000000000000000000000000000000000000000005", value = 24}
 ]
 0x6AF0674791925f767060Dd52f7fB20984E8639d8 = [
-     {key = "0x0000000000000000000000000000000000000000000000000000000000000005", value = 8}
+     {key = "0x0000000000000000000000000000000000000000000000000000000000000005", value = 10}
```

Comment:
yes, two tx a minute apart on this Safe: https://sepolia.etherscan.io/address/0x6af0674791925f767060dd52f7fb20984e8639d8#events